numpy/core/tests/test_umath.py

```diff
@@ -1668,7 +1668,7 @@ def test_tanh(self):
         for dt in ['e', 'f', 'd']:
             in_arr = np.array(in_, dtype=dt)
             out_arr = np.array(out, dtype=dt)
-            assert_equal(np.tanh(in_arr), out_arr)
+            assert_array_max_ulp(np.tanh(in_arr), out_arr, 3)
```

Comment:
Any reason for generalize this change among all the fp datatypes?

Reply:
tanh for float32 and float64 uses the SVML algorithm (one you converted to universal intrinsic), which also has a max ULP error of 3.

Reply:
Ah ok, I thought you caught new errors.